src/Pages/Platforms/currentConditions.tsx

```diff
@@ -1,7 +1,7 @@
 /**
  * Display current conditions for selected platform.
  */
-import * as React from "react"
+import React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)